spec/command-t/vim_spec.rb

```diff
@@ -11,4 +11,12 @@
       expect(CommandT::VIM.escape_for_single_quotes(input)).to eq(expected)
     end
   end
+
+  describe '.wildignore_to_regexp' do
+    it 'properly matches directories at any level' do
+      regexp = Regexp.new(CommandT::VIM.wildignore_to_regexp('*/foo/*'))
+      expect(regexp).to match('some/path/to/foo/file.js')
+      expect(regexp).to_not match('some/path/to/foo')
```

Comment:
Yay for tests!

And this really highlights the difficulty of translating `'wildignore'` patterns to a regular expression: `'wildignore'` patterns care about [the semantics of whether something is a file or a directory](https://github.com/wincent/command-t/blob/915048e6aed264dd7bf68e29ba485ade4c44f845/doc/command-t.txt#L1120-L1135), but regexes only care if text matches a pattern. Oh well!